Database/QuestieDB.lua

```diff
@@ -124,6 +124,7 @@ local questTagCorrections = {
     [90190] = {1, "Elite"},
     [90202] = {1, "Elite"},
     [90204] = {1, "Elite"},
+    [90211] = {1, "Elite"},
```

Comment:
Since you do not need to fight the Scout we shouldn't mark it as an Elite quest.

```suggestion
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5609/commits/82f0883c19e082887899e9a1192485228c484edf